catalog/imaging/catalog-video-transformer/src/main/java/com/connexta/transformer/video/ListOfDoublesHandler.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+import org.codice.ddf.libs.klv.KlvDataElement;
+import org.codice.ddf.libs.klv.data.numerical.KlvIntegerEncodedFloatingPoint;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Attribute;
+
+/**
+ * This handler expects floating point data and generates a list of Doubles.
+ */
+class ListOfDoublesHandler extends BaseKlvHandler {
```

Comment:
There are a number of `ListOfXHandler`s that are almost identical. I think it's possible to create a generic class that can do what this set of classes does. See [this gist](https://gist.github.com/jrnorth/f3a47702ebe494dde155f857634ed234) for an option.